Sources/Options.swift

```diff
@@ -178,6 +178,36 @@ public enum ClosureVoidReturn: String, CaseIterable {
     case preserve
 }
 
+public enum TrailingCommas: CaseIterable, RawRepresentable {
+    case inline
+    case always
+    case collections
+
+    public init?(rawValue: String) {
+        switch rawValue {
+        case "always", "true":
+            self = .always
+        case "inline", "false":
+            self = .inline
+        case "collections":
+            self = .collections
+        default:
+            return nil
+        }
+    }
+
+    public var rawValue: String {
+        switch self {
+        case .inline:
+            return "inline"
+        case .always:
+            return "inline"
+        case .collections:
+            return "collections"
+        }
+    }
+}
```

Comment:
Instead of using a custom `RawRepresentable` conformance, maybe we should use the `altOptions` `OptionDescriptor` initializer:

```swift
    let trailingCommas = OptionDescriptor(
        argumentName: "commas",
        displayName: "Commas",
        help: "Trailing commas: \"always\" (default), \"inline\", or \"collections\"",
        keyPath: \.trailingCommas,
        altOptions: [
          "true": .always,
          "false": .inline,
        ]
    )
```

```suggestion
public enum TrailingCommas: String, CaseIterable {
    case inline
    case always
    case collections
}
```

Reply:
Although this won't be necessary if we rework the option: https://github.com/nicklockwood/SwiftFormat/pull/2081#discussion_r2136176729